config/images.yml

```diff
@@ -220,6 +220,11 @@
   variants:
     - "static/images/content/blog/thumbnails/abigail-beeley.jpg"
 
+  "static/images/content/blog/abigail-beeley-crop.jpg":
```

Comment:
Just a reminder that this change will need removing from this PR before it is merged @spencerldixon @jenhadfield-dfe !

In addition, the image should also be removed from `/images/content/blog` - if it is needed when we actually start using this component, it can be added back in (without the surname)